### DISCUSSION TITLE
dubbo triple如何兼容grpc的ping

### DISCUSSION THREAD

**Discussion by Webster-Yang** [apache/dubbo#15009] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java Samples (apache/dubbo-samples)

### Details

grpc通过ping来保活
triple协议完全兼容grpc
那么dubbo triple协议如何兼容grpc的ping

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment, marked as answer]
`Ping` is a standard request in HTTP/2
